src/components/Header/index.tsx

```diff
@@ -9,11 +9,11 @@ import Nav from "./Nav";
 
 export default function Header() {
   return (
-    <header className="h-header sticky top-0 z-[20] flex w-full items-center justify-center bg-background-primary backdrop-blur-xl">
-      <div className="flex w-full max-w-screen-xl flex-col gap-2 p-4 pb-2 md:pb-4">
+    <header className="sticky top-0 z-[20] flex h-header w-full items-center justify-center bg-background-primary backdrop-blur-xl">
```

Comment:
![Screenshot 2025-05-09 at 11 39 37](https://github.com/user-attachments/assets/6a3a1adc-f362-492f-9846-91827f090266)

Casues padding issues. Tbh we can leave it as it was i think, the few px less wide is not a big deal for the max width